android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1620,6 +1622,20 @@ public void openBraveContentFilteringSettings() {
         settingsLauncher.startSettings(this, ContentFilteringFragment.class);
     }
 
+    public boolean isNightlyModeEnabled() {
```

Comment:
nit: `NightlyMode` sounds like it has to do with the release channel, maybe `NightMode` instead?